src/e2e-test/features/DesignTimeValidation.feature

```diff
@@ -0,0 +1,199 @@
+#
+# Copyright © 2023 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+#
+
+@Oracle
+Feature: Oracle - Verify Oracle source plugin design time validation scenarios
+
+  @Required
+  Scenario: To verify validation message when user provides invalid Host
+    Given Open DataFusion Project with replication to configure replication job
+    When Enter input plugin property: "name" with pipelineName
+    And Click on the "Next" button in replication to navigate
+    And Select Source plugin: "Oracle" from the replication plugins list
+    Then Select radio button plugin property: "connectivityMethod" with value: "private-connectivity"
```

Comment:
Nit: Use camel case for variable 'private-connectivity'

Reply:
It is the value to be selected.